src/parkapi_sources/converters/apcoa/mapper.py

```diff
@@ -0,0 +1,83 @@
+"""
+Copyright 2024 binary butterfly GmbH
+Use of this source code is governed by an MIT-style license that can be found in the LICENSE.txt.
+"""
+
+from datetime import datetime, timezone
+
+from parkapi_sources.models import StaticParkingSiteInput
+from parkapi_sources.models.enums import PurposeType
+
+from .validators import ApcoaNavigationLocationType, ApcoaOpeningHoursWeekdays, ApcoaParkingSiteInput, ApcoaParkingSpaceType
+
+
+class ApcoaMapper:
+    @staticmethod
+    def map_static_parking_site(apcoa_input: ApcoaParkingSiteInput) -> StaticParkingSiteInput:
+        static_parking_site_input = StaticParkingSiteInput(
+            uid=str(apcoa_input.CarParkId),
+            name=apcoa_input.CarparkLongName if apcoa_input.CarparkLongName else apcoa_input.CarparkShortName,
+            lat=next(
```

Comment:
1) You do the same iteration two times. Why not do it just once?
2) When there is no entry with`ApcoaNavigationLocationType.DEFAULT`, you will have an exception for the whole converter which is unwanted

Reply:
Oh that's right, i have now removed the second iteration and checked for no entry in LocationType